src/postamble.js

```diff
@@ -474,7 +466,7 @@ function exit(status, implicit) {
     ABORT = true;
   }
 
-  quit_(status, new ExitStatus(status));
+  quit_(status, new ExitStatus(status), keepRuntimeAlive());
```

Comment:
how about avoiding the `keepRuntimeAlive()` call here and on line 211, and instead doing it in the called function? That seems simpler too, as we always pass the same thing?